### PR TITLE
Nav Bar, preliminary

### PR DESCRIPTION
**Pull request by catmcclelland**
- Nav bar with logo (clicking leads to home), feed button (leads to home as well), topics mega menu (will need to eventually fill this with discussion topics for users to explore), and conditionally rendered account menu OR login / sign up buttons. 
- It is mobile responsive
- It shows which page you are currently on
- Not shown but when mobile nav is closed it has a hamburger menu icon :)
- There are some linting and tsc errors I couldn't figure out at the moment but non-breaking
Screenshot 2023-07-06 at 4 08 54 PM
Screenshot 2023-07-06 at 4 10 21 PM
Screenshot 2023-07-06 at 4 11 29 PM

Screenshot 2023-07-06 at 4 08 45 PM
Screenshot 2023-07-06 at 4 08 21 PM

### CODE REVIEW

File: src/pages/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import React from 'react'
-
 const HomePage = (): JSX.Element => {
   return (
-        <div>This is the home page!</div>
+    <>
```
Comment (jsmithb117) [inline src/pages/HomePage/HomePage.tsx:3]: Fragments are only required when you're rendering multiple React components without an outer HTML element.

**Review by jsmithb117 — COMMENTED** [review]
Wow, that looks like a lot of work!  

I found a few issues

- Typescript allows us to type all of the props, so I don't think prop-types is necessary, is it?

- Props are being passed improperly for NavList

- Typescript index error for colors

**Review by jsmithb117 — APPROVED** [review]
LGTM

File: src/pages/FeedPage/FeedPage.tsx [hunk 2]
```diff
@@ -1,6 +1,65 @@
...
+
+  function handleClick (): void {
+    setCreateDebate((prevCreateDebate) => !prevCreateDebate)
+    if (setMessages) {
+      setMessages([])
+    }
+  }
+  function getPersona (index: number): Persona | undefined {
+    if (personas) {
+      if (index === 0 || index % 2 === 0) {
```
Comment (Elizabeth-Giovanniello) [inline src/pages/FeedPage/FeedPage.tsx:26]: nit: do you need this or statement? is 0 % 2 not also 0?

**Pull request merged**